.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,7 +11,26 @@ labels: 'type:bug, status:requirements, priority-5-triage'
 
 **What Renovate type, platform and version are you using?**
 
-<!-- Tell us if you're using the hosted App, or if you are self-hosted Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
+<!-- Tell us if you're using the hosted App, or if you are self-hosting Renovate yourself. Platform too (GitHub, GitLab, etc) plus which version of Renovate if you're self-hosted. -->
+
+**Renovate type**
+
+- [ ] Hosted app
+- [ ] Self hosted
+
+**Renovate platform**
+
+- [ ] Azure DevOps
```

Comment:
```suggestion
- [ ] Azure DevOps (dev.azure.com)
```

I'll use the same part from https://github.com/renovatebot/renovate/blob/main/docs/usage/modules/platform.md here as well, so we're consistent in docs and in the bug report template.